pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.pkgproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <Version>2.0.0</Version>
```

Comment:
I picked 2.0.0 because this is intended to first work with NS2.0. Is that sufficient reasoning?